app/src/main/java/com/sopt/dive/feature/home/components/ProfileComponents.kt

```diff
@@ -0,0 +1,81 @@
+package com.sopt.dive.feature.home.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+
+@Composable
+fun MyProfileComponent(userId: String, userNickname: String) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(Color(0xFFE0F7FA))
+            .padding(16.dp)
+    ) {
+        Column {
+            Text(text = "내 프로필", fontWeight = FontWeight.Bold, color = Color.Black)
+            Text(text = userNickname, color = Color.Black)
+            Text(text = "ID: $userId", color = Color.Gray)
+        }
+    }
+}
+
+@Composable
+fun FriendProfileComponent(name: String, statusMessage: String, hasMusic: Boolean) {
```

Comment:
여기 매개변수로 Modifier를 받는 게 더 유연? 할 것 같아요! 아래는 modifier = modifer 
```suggestion
fun FriendProfileComponent(name: String, statusMessage: String, hasMusic: Boolean, modifier: Modifier = Modifier  ) {
```

Reply:
혹시 왜 그렇게해야 유연한지 간단하게 설명해주실 수 있을까용 ?? 궁금해서 여쭤봅니다 !!

Reply:
외부에서 컴포넌트를 사용하는 쪽에서 상황에 맞게 스타일 조정이 가능해져 재사용성이 향상됩니다!